src/domain/occurrence/eventOccurrenceForm/eventOccurrenceForm.module.scss

```diff
@@ -52,7 +52,7 @@
 
   .formActions {
     display: flex;
-    margin: $largeMargin 0;
+    margin-top: $largeMargin 0;
```

Comment:
I guess this should be 
`margin-top: $largeMargin;`

Reply:
yes :) Will fix later